### ISSUE TITLE
Slightly fade out read tweets and "mark all as read"

### ISSUE THREAD

**Issue by oktayacikalin** [baedert/corebird#122]
Currently it's not easy to spot new tweets. It would be nice to have something like "mark all as read" and slightly fade out read tweets.

**Comment by tglman** [comment]
i need the same, one easy way to mark tweets as read could be just go over with the mouse, as well as mark as read all the tweet on the current "page" after a while (~10sec of even 3sec :D ) that the window is focused

will be cool also update the title with the unread tweet count like "@tglman (3)"
@baedert what do you think about this, do you accept contributions on this ?

**Comment by baedert** [comment]
Why are you not just scrolling up? Having any sort of timeout seems pointless since we are already doing the scrolling based approach (of course there's no indication of how many unread tweets there are, that would be #106).

**Comment by tglman** [comment]
yep @baedert i'm using scrolling up, but sometime i scroll too fast and i loose a bit the last read point :), also a marker line just before last read tweet can be usefull, anyway i'll start to play with the count, i'll comment it in the #106 ;)
